source/modules/task-list.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * @name addTaskButton
+ * @function
+ * @description Opens or closes add-task form
+ */
+function addTaskButton() {
+    let button = document.getElementById("add-task-form");
+    if (button.classList.contains("hidden") === true) {
+        button.classList.remove("hidden");
+    }
+    else {
+        button.classList.add("hidden");
+    }
+}
+
+/**
+ * @name cancelTask
+ * @function
+ * @description Closes add-task form
+ */
+function cancelTask() {
+    document.getElementById("task-name").value = "";
+    document.getElementById("add-task-form").classList.add("hidden");
+}
+
+/**
+ * @name saveTask
+ * @function
+ * @description Adds new task to task list
+ */
+function saveTask() {
+    let taskNameInput = document.getElementById("task-name");
+    let taskList = document.getElementById("task-list");
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/cse112-sp22-group5/cse112-sp22-group5/pullRequest?prid=9407153)
```suggestion
    let taskList = document.getElementById('task-list');
```